packages/@coorpacademy-progression-engine/package.json

```diff
@@ -13,7 +13,8 @@
     "ava": "cross-env NODE_ENV=test ava",
     "nyc": "cross-env NODE_ENV=test nyc npm run ava",
     "test": "npm run lint && npm run flow && npm run nyc",
-    "flow": "flow",
+    "killflow": "pkill flow",
```

Comment:
Why do you need to kill it ? It will restart automatically if it's not already launched

Reply:
removed